service/src/test/java/com/epam/rft/atsy/service/passwordchange/validation/impl/PasswordAllFieldFilledRuleTest.java

```diff
@@ -0,0 +1,144 @@
+package com.epam.rft.atsy.service.passwordchange.validation.impl;
+
+import com.epam.rft.atsy.service.domain.PasswordChangeDTO;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PasswordAllFieldFilledRuleTest {
+
+    //A password is considered blank if it contains only whitespace characters.
+    public static final String BLANK_PASSWORD = System.getProperty("line.separator");
```

Comment:
This will only test the line separator of the current OS, so on differenct systems the tests will test a slightly different thing. Please specifiy the line separators directly.

Also, there could be other kinds of blank passwords like empty or contains spaces only. Because of this, it would be a good idea to make this a parameterized test, and move out the success case to a different (non-parameterized) test class, so it won't run unnecessarily multiple times.